source/linux/Foundational_Components/Kernel/Kernel_Drivers/DM-Timer.rst

```diff
@@ -0,0 +1,152 @@
+.. include:: /replacevars.rst.inc
+
+########
+DM TIMER
+########
+
+.. rubric:: DM Timer Driver Overview
+
+- This guide covers a general overview on DM Timer.
+  It supports timers found on TI's |__PART_FAMILY_NAME__| SoCs.
+
+- Each timer contains a free-running upward counter with autoreload capability on overflow.
+  Each timer includes compare logic to allow an interrupt event on a programmable counter
+  matching value. A dedicated output signal can be pulsed/toggled on an overflow or a match
+  event. This offers time-stamp trigger signaling or PWM signal sources. A dedicated input
+  signal can be used to trigger an automatic timer counter capture or an interrupt event on a
+  programmable input signal transition.
+
+- Each timer comes with pins that can be used as Timer Capture interrupts
+  or as timer PWM outputs.
+
+- For more detailed information on timers, please refer to the:
+  Peripherals/Timer Modules/Timers section of the device specific TRM.
+
+|
```

Comment:
Do not use these to artificially pad sections. They are naturally separated based on the sphinx builder.